src/gui/advancedsettings.cpp

```diff
@@ -418,7 +421,9 @@ void AdvancedSettings::loadAdvancedSettings()
     // Download tracker's favicon
     cb_tracker_favicon.setChecked(mainWindow->isDownloadTrackerFavicon());
     addRow(DOWNLOAD_TRACKER_FAVICON, tr("Download tracker's favicon"), &cb_tracker_favicon);
-
+    // Save path history length
+    spinDownloadPathHistoryLength.setValue(AddNewTorrentDialog::savePathHistoryLength());
+    addRow(DOWNLOAD_PATH_HISTORY_LENGTH, tr("Download path history length"), &spinDownloadPathHistoryLength);
```

Comment:
Download path: I wonder if this should be called "Save path" as this term is used in options & add torrent dialog.

Reply:
Done.